dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -567,47 +529,23 @@ def validate_job(self, request, interfaces):
             job_type_name = job.job_type.name
         else:
             validation_error_type = ValidationError.jobError
-            writeFileError(job_id, self.filename, validation_error_type)
+            writeFileError(job_id, None, validation_error_type)
             raise ResponseException(
                 'Job ID {} is not a validation job (job type is {})'.format(job_id, job.job_type.name),
                 StatusCode.CLIENT_ERROR, None,
                 validation_error_type)
 
-        # todo: remove the following try/catch once 1st batch of changes are merged
-        try:
-            jobTracker.markJobStatus(job_id, "running")
-            if job_type_name == 'csv_record_validation':
-                self.runValidation(job_id, interfaces)
-            elif job_type_name == 'validation':
-                self.runCrossValidation(job_id, interfaces)
-            else:
-                raise ResponseException("Bad job type for validator",
-                    StatusCode.INTERNAL_ERROR)
-
-            return JsonResponse.create(StatusCode.OK, {"message":"Validation complete"})
-        except ResponseException as e:
-            _exception_logger.exception(str(e))
-            self.markJob(job_id, jobTracker, "invalid", self.filename,e.errorType, e.extraInfo)
-            return JsonResponse.error(e, e.status)
-        except ValueError as e:
-            _exception_logger.exception(str(e))
-            # Problem with CSV headers
-            exc = ResponseException(str(e),StatusCode.CLIENT_ERROR,type(e), ValidationError.unknownError) #"Internal value error"
-            self.markJob(job_id,jobTracker, "invalid", self.filename, ValidationError.unknownError)
-            return JsonResponse.error(exc, exc.status)
-        except Error as e:
```

Comment:
It doesn't look like this CSV error is being explicitly caught anymore, which I think means that the job would be marked "failed" instead of "invalid" now. Maybe not a problem?